pkg/pymetadata/parse.go

```diff
@@ -9,9 +9,6 @@ import (
 	"golang.org/x/xerrors"
 )
 
-// Parse parses METADATA file for library Name and Version
-// currently does not parse its possible dependencies
-// https://packaging.python.org/specifications/core-metadata/#requires-dist-multiple-use
 func Parse(r io.Reader) ([]types.Library, error) {
```

Comment:
We can return a single value.

```suggestion
func Parse(r io.Reader) (types.Library, error) {
```